### ISSUE TITLE
Why does newDeconv call the model again

### ISSUE THREAD

**Issue by harveyslash** [akikan/Visual_attribute_transfer_deep_image_analogy_tensorflow#1]
Why is the model features called after deconvolution is done ? 
Shouldn't it be the same if the deconv is done for one layer above?

**Comment by akikan** [comment]
oops! You are right. I referred two sources when I written this program. Maybe it is the cause.
I will fix it later.

**Comment by harveyslash** [comment]
I am trying to get a pytorch version of this to get working, but it doesnt work. Do you think you can take a look at it?  I will be very grateful, and will give due credit.

**Comment by harveyslash** [comment]
At the moment, this is my output,
![unknown-1](https://user-images.githubusercontent.com/7107410/32139102-8e45da98-bc0e-11e7-9715-cc1302052da2.png)

I ran your code, and it produces much smoother results. 
I am also using adam , and learning rate of 1 , 1000 epochs

**Comment by akikan** [comment]
I have never used pytorch.
So it takes some time to make it.
If you still will wait, let's do my best.

**Comment by harveyslash** [comment]
is there anywhere I can have a quick chat with you ? i have implemented most of the logic , i just need to confirm one or two things. Help is greatly appreciated

**Comment by akikan** [comment]
sorry
Even if I can reply to chat it will be very late.
But please do listen when you want my help.

thank you
